src/db/seeders/20220514012040-sample-schools.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    return await queryInterface.bulkInsert('schools', [
+      {
+        name: 'Dulwich College Shanghai Pudong',
+        timezone: 'Asia/Shanghai',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: 'Dulwich College Singapore',
+        timezone: 'Asia/Singapore',
```

Comment:
Can you create a new type or interface called Timezone and also a Constant to store all the available timezones?

Reply:
okok good thinking